repos/system_upgrade/common/models/installedtargetkernelversion.py

```diff
@@ -10,3 +12,39 @@ class InstalledTargetKernelVersion(Model):
     """
     topic = SystemInfoTopic
     version = fields.String()
+
+
+class KernelInfo(Model):
+    """
+    Information about the booted kernel.
+    """
+    topic = SystemInfoTopic
+
+    pkg = fields.Model(RPM)
+    """ Package providing the booted kernel. """
+
+    uname_r = fields.String()
+    """``uname -r`` of the booted kernel."""
+
+    type = fields.StringEnum(['ordinary', 'realtime'], default='ordinary')
+    # @FixMe(mhecko): I want to use kernel_lib.KernelType here, but I cannot import any library code (yet).
+    # #               Figure out how to do it.
+
+
+class InstalledTargetKernelInfo(Model):
+    """Information about the installed target kernel."""
+    topic = SystemInfoTopic
+
+    pkg_nevra = fields.String()
+    """Name, epoch, version, release, arch of the target kernel package."""
+
+    # No one uses this information yet and scanning it it costly if running in debug (large
+    # RPM provides of kernel) flood stdout
+    # uname_r = fields.String()
+    # """Kernel release of the target kernel."""
```

Comment:
@MichalHe you can specify our logger for the run command, so the output is hidden / skipped. e.g.
```
run(['rpm', '-q', '--provides', kernel_pkgname], callback_raw=lambda fd, value: None, callback_linebuffered=lambda fd, value: None)
```

Reply:
I agree with you (it's better to add more when needed), just wanted to share how to hack it in case of need :)